### PR TITLE
Cleanup typo

### PR DESCRIPTION
**Pull request by juliamrch**
Replaced "nomber" by "number" lol

### CODE REVIEW

File: partials/ssh-keys.md [hunk 1]
```diff
@@ -68,19 +69,27 @@ You may already have an SSH key and so do not need to generate a new one. To che
 1. If you don't have it, download [Git for Windows](https://git-for-windows.GitHub.io/) and install it.
 2. Run **Git Bash** (from the *Start Menu* or from the *Explorer* with the contextual menu (right click)).
 3. Run `cd ~/.ssh/` in your Terminal.
-4. If the folder exists, run `ls` and check if a pair of key exists : *id_ed25519* and *id_ed25519.pub* or *id_rsa* and *id_rsa.pub*. We would recommend using *ed25519* keys. Smaller to copy and way stronger than 2048-bit RSA keys. If you can find them, you do not need to generate a new one, simply go to the following "Add your key on Clever Cloud" part!
+3. If the folder exists, run `ls` and check if a pair of key exists : `id_ed25519` and `id_ed25519.pub`.
```
Comment (Bencaddyro) [inline partials/ssh-keys.md:72]:
```suggestion
4. If the folder exists, run `ls` and check if a pair of key exists : `id_ed25519` and `id_ed25519.pub`.
```

File: partials/create-application.md [hunk 2]
```diff
@@ -2,18 +2,15 @@
...
+1. Make sure you have clever-tools installed locally or follow our [CLI getting started]({{< ref "/reference/clever-tools/getting_started.md" >}}) guide.
+2. In your code folder, do `clever create --type <type> <app-name> --region <zone> --org <org>` where :
+   1. `type` is the type of technology you rely on
+   2. `app-name` the name you want for your application,
+   3. `zone` deployment zone (`par` for Paris and `mtl` for Montreal)
+   4. `org` the organization ID the application will be created under.
 
-Refer to the [documentation]({{< ref "/reference/clever-tools/create.md" >}}) for more details on application creation with Clever Tools.
\ No newline at end of file
+Refer to [clever create]({{< ref "/reference/clever-tools/create.md" >}}) for more details on application creation with Clever Tools.
```
Comment (sebartyr) [inline partials/create-application.md:16]:
```suggestion
Refer to [clever create]({{< ref "/reference/clever-tools/create.md" >}}) for more details on application creation with Clever Tools.
```

Comment (sebartyr) [inline partials/create-application.md:16]:
```suggestion
Refer to [clever create]({{< ref "/reference/clever-tools/create.md" >}}) for more details on application creation with Clever Tools.ff
```

Comment (sebartyr) [inline partials/create-application.md:16]:
```suggestion
Refer to [clever create]({{< ref "/reference/clever-tools/create.md" >}}) for more details on application creation with Clever Tools.gg
```

File: partials/deploy-git.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
 ## Git Deployment on Clever Cloud
 
-*You need git on your computer to deploy via this tool. Here is the official website of Git to get more
-information: [git-scm.com](https://git-scm.com)*
+*You need git on your computer to deploy via this tool. Here is the official website of Git to get more information: [git-scm.com](https://git-scm.com)*
```
Comment (sebartyr) [inline partials/deploy-git.md:3]:
```suggestion
*You need Git on your computer to deploy via this tool. Here is the official website of Git to get more information: [git-scm.com](https://git-scm.com)*
```

File: partials/deploy-git.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 
 ### Setting up your remotes
 
-1. The "Information" page of your app gives you your git deployment URL. It looks like this:
-``git+ssh://git@push.clever-cloud.com/<your_app_id>.git``. Copy it in your clipboard.
+1. The "Information" page of your app gives you your git deployment URL, it looks like this:
```
Comment (sebartyr) [inline partials/deploy-git.md:7]:
```suggestion
1. The "Information" page of your app gives you your Git deployment URL, it looks like this:
```

File: partials/language-specific-deploy/go.md [hunk 5]
```diff
@@ -6,9 +6,9 @@ Be sure that:
 
 * Your application listens to the wild network **0.0.0.0**, not only `localhost` or `127.0.0.1`
 * Your application listens on port **8080**
-* You put your main code in a file named `main.go` (if you do not do that, Go will generate a library and not an executable)
+* You put your main code in a file named `main.go` (If you do not do that, Go will generate a library and not an executable)
```
Comment (sebartyr) [inline partials/language-specific-deploy/go.md:9]:
```suggestion
* You put your main code in a file named `main.go` (if you do not do that, Go will generate a library and not an executable)
```

I dont think upper is useful here

File: partials/language-specific-deploy/php.md [hunk 6]
```diff
@@ -154,31 +154,31 @@ We use Apache 2 as HTTP Server. In order to configure it, you can create a `.hta
...
+
 However, directives who applies to the entire application must be declared in a `.htaccess` file to the application root.
 
 ### htpasswd
 
-If you need basic authentication, you can use the `.htpasswd` file. The path to the `.htpasswd` of the `AuthUserFile` directive has to be absolute. Your site root folder is available at `/var/www/bas/site/`, so the directive should look like:
-`AuthUserFile /var/www/bas/site/.htpasswd`
+If you need basic authentication, you can use the `.htpasswd` file. The path to the `.htpasswd` of the `AuthUserFile` directive has to be absolute.
+
+Your site root folder is available at `/var/www/bas/site/`, so the directive should look like: `AuthUserFile /var/www/bas/site/.htpasswd`
```
Comment (sebartyr) [inline partials/language-specific-deploy/php.md:164]:
```suggestion
```

File: partials/language-specific-deploy/php.md [hunk 7]
```diff
@@ -154,31 +154,31 @@ We use Apache 2 as HTTP Server. In order to configure it, you can create a `.hta
...
 **By default, the HTTP timeout is se to 3 minutes (180 seconds)**.
 
 ### Force HTTPS traffic
 
-Load balancers handle HTTPS traffic ahead of your application. You can use the
-`X-Forwarded-Proto` header to know the original protocol (`http` or `https`).
+Load balancers handle HTTPS traffic ahead of your application. You can use the `X-Forwarded-Proto` header to know the original protocol (`http` or `https`).
 
 Place the following snippet in a `.htaccess` file to ensure that your visitors
 only access your application through HTTPS.
```
Comment (sebartyr) [inline partials/language-specific-deploy/php.md:179]:
```suggestion
Place the following snippet in a `.htaccess` file to ensure that your visitors only access your application through HTTPS.
```

File: partials/language-specific-deploy/php.md [hunk 8]
```diff
@@ -333,40 +332,39 @@ The following is the list of tested CMS by our team.
...
+</table>
 
 ## Available extensions and modules
 
-You can check enabled extensions and versions by viewing our `phpinfo()` example for
+You can check enabled extensions and versions by viewing our `phpinfo()` example for:
+
 - [PHP 5.6](https://php56info.cleverapps.io).
 - [PHP 7.2](https://php72info.cleverapps.io).
 - [PHP 7.3](https://php73info.cleverapps.io).
```
Comment (sebartyr) [inline partials/language-specific-deploy/php.md:370]:
```suggestion
- [PHP 7.3](https://php73info.cleverapps.io).
- [PHP 8.1](https://php81info.cleverapps.io).
```

File: partials/language-specific-deploy/python.md [hunk 9]
```diff
@@ -172,14 +173,16 @@ We also support celery apps out of the box. To deploy a celery app, use the `CC_
...
 
 You can also activate beat with `CC_PYTHON_CELERY_USE_BEAT=true` and provide a given log dir for celery with `CC_PYTHON_CELERY_LOGFILE="/path/to/logdir"`.
 
 The `CC_PYTHON_CELERY_LOGFILE` path is relative to the application's path.
 
 {{< alert "warning" "Beware of timezones with Celery Beat!" >}}
-   There is a bug in recent versions of Celery. You need to add the `CELERY_TIMEZONE = 'UTC'` environment variable. The bug is documented here: [https://GitHub.com/celery/celery/issues/4184](https://GitHub.com/celery/celery/issues/4184).
+There is a bug in recent versions of Celery. You need to add the `CELERY_TIMEZONE = 'UTC'` environment variable.
+   
+The bug is documented here: [https://GitHub.com/celery/celery/issues/4184](https://GitHub.com/celery/celery/issues/4184).
```
Comment (sebartyr) [inline partials/language-specific-deploy/python.md:187]:
```suggestion
```

this issue is now closed

File: partials/language-specific-deploy/php.md [hunk 10]
```diff
@@ -154,31 +154,27 @@ We use Apache 2 as HTTP Server. In order to configure it, you can create a `.hta
...
 The `.htaccess` file can be created everywhere in you app, depending of the part of the application covered by directives.
+
 However, directives who applies to the entire application must be declared in a `.htaccess` file to the application root.
 
 ### htpasswd
 
-If you need basic authentication, you can use the `.htpasswd` file. The path to the `.htpasswd` of the `AuthUserFile` directive has to be absolute. Your site root folder is available at `/var/www/bas/site/`, so the directive should look like:
-`AuthUserFile /var/www/bas/site/.htpasswd`
 
 Alternatively, you can configure basic authentication using [environment variables]({{< ref "reference/reference-environment-variables.md#php" >}}). You will need to set `CC_HTTP_BASIC_AUTH` variable to your own `login:password` pair. If you need to allow access to multiple users, you can create additional environment `CC_HTTP_BASIC_AUTH_n` (where `n` is a number) variables.
```
Comment (Bencaddyro) [inline partials/language-specific-deploy/php.md:163]:
```suggestion
You can configure basic authentication using [environment variables]({{< ref "reference/reference-environment-variables.md#php" >}}). You will need to set `CC_HTTP_BASIC_AUTH` variable to your own `login:password` pair. If you need to allow access to multiple users, you can create additional environment `CC_HTTP_BASIC_AUTH_n` (where `n` is a number) variables.
```

**Review by Bencaddyro — APPROVED** [review]

**Pull request merged**